force-app/main/default/classes/newTaskTraining.cls

```diff
@@ -0,0 +1,18 @@
+public class newTaskTraining {
+    public static void createTask() {
+        Date today = Date.today();
+        List<Training__c> endedTrains = [SELECT Name, End_Date__c, Training_Course__r.OwnerId
+                                        FROM Training__c
+                                        WHERE DAY_ONLY(End_Date__c) = :today];
+        //System.debug(endedTrains);
+        List<Task> taski = new List<Task>();
+        for(Training__c train :endedTrains) {
```

Comment:
Please use “training” instead of “train”, because “train” is an existing English word and it is considered as a better code  style to use full variable names